test/tools/obdscanner.uts

```diff
@@ -259,30 +253,26 @@ s1_pid03 = OBD()/OBD_S01_PR(data_records=[OBD_S01_PR_Record()/OBD_PID03(fuel_sys
 s1_pid0B = OBD()/OBD_S01_PR(data_records=[OBD_S01_PR_Record()/OBD_PID0B(data=100)])
 s1_pid0F = OBD()/OBD_S01_PR(data_records=[OBD_S01_PR_Record()/OBD_PID0F(data=50)])
 s1_queue.put(s1_pid00)
-s1_queue.put(s1_pid0F)
```

Comment:
Why are you changing the order and the next results?

Reply:
I tried fixing the tests, apparently the order of the messages does matter for this test.
I now comited a more robust version of this, were the order is not relevant anymore.